spec/services/csv_service_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe CsvService do
+  subject { described_class.new }
+
+  before(:each) do
+    investment_params = RedmineRequestCsv::INVESTMENT_PARAMS
+
+    @url_investment_time = RedmineRequestCsv.new(investment_params).instance_variable_get('@url')
+    @url_total_time = RedmineRequestCsv.new.instance_variable_get('@url')
+
+    http_response_total_time = "User,Total time\nMax Muller,8.00\nHans Meier,12.00\nTotal time,20.00"
+    stub_request(:any, @url_total_time).to_return(body: http_response_total_time)
+  end
+
+  describe '#time_entries' do
+    context 'investment time is taken' do
+      it 'returns the time entries' do
```

Comment:
"returns the time entries" doesn't tell me a lot. What happens? What is expected?